src/applications/disability-benefits/all-claims/pages/privateMedicalRecords.js

```diff
@@ -78,6 +79,26 @@ export const uiSchema = {
         _.get(DATA_PATHS.hasPrivateRecordsToUpload, data, false),
     },
   ),
+  'view:patientAcknowledgement': {
+    'ui:title': ' ',
+    'ui:help': patientAcknowledgmentText,
+    'ui:options': {
+      expandUnder: 'view:uploadPrivateRecordsQualifier',
+      expandUnderCondition: data =>
+        _.get('view:hasPrivateRecordsToUpload', data, false) === false,
+      showFieldLabel: true,
```

Comment:
memory's a little fuzzy on this one, but did you verify that you need `showFieldLabel` for the headings to show correctly??

Reply:
Yeah, the `showFieldLabel` was needed to show the empty title so the validation error message would take appear. I've filed a ticket to USFS for this.